configure.ac

```diff
@@ -8077,6 +8077,7 @@ PY_STDLIB_MOD([_gdbm],
  PY_STDLIB_MOD([readline],
   [], [test "$with_readline" != "no"],
   [$READLINE_CFLAGS], [$READLINE_LIBS])
+PY_STDLIB_MOD([_remotedebugging], [test "$TEST_MODULES" = yes])
```

Comment:
@pablogsal should we keep this `"$TEST_MODULES" = yes` condition? It's inherited from before the rename, but I don't know what conditions you want the module to be enabled under.

Reply:
Nop, that was from before when this was a test module. Let's remove it

Reply:
Ok, I've pushed a fix for this & also the rename to `_remote_debugging`. Thanks!